src/Raven.Client/Documents/Subscriptions/AsyncDocumentSubscriptions.cs

```diff
@@ -26,37 +26,37 @@ public AsyncDocumentSubscriptions(IDocumentStore store)
             _store = store;
         }
 
-        public Task<long> CreateAsync<T>(SubscriptionCreationOptions<T> options, string database = null)
+        public Task<string> CreateAsync<T>(SubscriptionCreationParams<T> subscriptionCreationParams, string database = null)
         {
-            if (options == null)
+            if (subscriptionCreationParams == null)
                 throw new InvalidOperationException("Cannot create a subscription if criteria is null");
             
             var nonGenericCriteria = new SubscriptionCriteria(_store.Conventions.GetCollectionName(typeof(T)))
             {
-                FilterJavaScript = options.Criteria.FilterJavaScript,
+                FilterJavaScript = subscriptionCreationParams.Criteria?.FilterJavaScript,
             };
 
-            var subscriptionCreationDto = new SubscriptionCreationOptions
+            var subscriptionCreationDto = new SubscriptionCreationParams
```

Comment:
@maximburyak Recently I've renamed it to Options, to be more consistent. Can we rename it to Options again?